### ISSUE TITLE
Parsing breakfast/lunch/dinner to datetime

### ISSUE THREAD

**Issue by alvinthen** [wit-ai/duckling#145]
Not sure if this is a bug or unintended feature.

Trying to understand `lunch` using duckling returns a time from 1200 to 1400, while breakfast/dinner returns nothing.

It would be great to be able to translate meals to a time range.

Tested on https://duckling.wit.ai/

**Comment by l5t** [comment]
Good idea. We can probably add it but one would argue that the time ranges for meal really depends on your country/culture so this is probably something that can be handled via a custom entity to detect the meal type, and then mapped to the specific time range for your app

**Comment by magnusburton** [comment]
@l5t I agree with @alvinthen tat we should add in breakfast and dinner time ranges. I did a quick search what it came up with was that [most countries tend to eat dinner in the 5.30pm to 9.00pm range](https://www.quora.com/What-is-the-typical-time-for-dinner-in-different-cities-around-the-world). A range for that period of time wouldn't hurt right? Compared to not returning anything at all.
